code/index.html

```diff
@@ -7,31 +7,33 @@
     <link rel="stylesheet" href="style.css" />
     <title>Technigo Project Cluedo</title>
   </head>
-  <body>
-    <section class="cards">
-      <article class="killer-deck" id="killerCard">
-        <p>KILLER</p>
+  <body >
+    <header>
+      <h2>CLUEDO GAME</h2>
```

Comment:
Maybe this will be suited as a `<h1>` instead? I can't see that you have one somewhere else :)